packages/nimble-components/src/button/tests/button.mdx

```diff
@@ -22,19 +22,97 @@ import * as buttonStories from './button.stories';
 
 <br />
 
+## Styling
+
+<br />
```

Comment:
We shouldn't be using `<br>` tags for effectively styling purposes like this. Can you create a tech debt task to create a stylesheet for our mdx files that can styles our headers as we need and remove all these `<br>` tags?

Reply:
Sure, here's the tech debt issue #1413 👍